wiki-service/src/main/java/org/exoplatform/wiki/service/impl/WikiServiceImpl.java

```diff
@@ -154,6 +154,7 @@ public WikiServiceImpl(ConfigurationManager configManager,
 
   @Override
   public void start() {
+    initTemplates();

```

Comment:
Yes, you are right, I met the same problem when creating new space. 
But when new space created, the function createWiki() will never be called. That's why this problem occurs.